docs/approvals.md

```diff
@@ -102,3 +102,131 @@ AN object containing the following keys:
   }
 }
 ```
+
+## getFormSubmissionApprovalById
+
+### Example
+
+```javascript
+const formSubmissionApproval = await approvals.getFormSubmissionApprovalById(
+  'aaaaaaaa-bbbb-4543-cccc-addddd543454',
+)
+// Use data here...
+```
+
+### Parameters
+
+AN object containing the following keys:
+
+| Parameter | Required | Type     | Description                            |
+| --------- | -------- | -------- | -------------------------------------- |
+| `id`      | Yes      | `string` | The id of the form submission approval |
+
+### Result (Resolved Promise)
+
+```json
+{
+  "formSubmissionMeta": {
+    "submissionId": "f1eadc2b-79c8-4f97-8d92-cde64b34911f",
+    "formId": 2,
+    "formsAppId": 1,
+    "dateTimeSubmitted": "2021-08-06T12:00:00.00Z"
+  },
+  "formApprovalFlowInstance": {
+    "id": 1,
+    "createdAt": "2021-08-06T12:00:00.00Z",
+    "formId": 2,
+    "submissionId": "f1eadc2b-79c8-4f97-8d92-cde64b34911f",
+    "approvalsFormsAppId": 1,
+    "steps": [
+      {
+        "group": "group1",
+        "label": "Step 1",
+        "isSkipped": false
+      }
+    ],
+    "status": "APPROVED",
+    "isLatest": true
+  },
+  "formSubmissionApproval": {
+    "group": "group1",
+    "formApprovalFlowInstanceId": 1,
+    "stepLabel": "Step 1",
+    "id": "a2fgdc5g-79c8-4f97-8d92-cde64b34956s",
+    "status": "APPROVED",
+    "createdAt": "2021-08-06T12:00:00.00Z",
+    "updatedAt": "2021-08-06T12:00:00.00Z"
+  },
+  "form": {
+    "id": 1,
+    "name": "testsform",
+    "formsAppEnvironmentId": 1,
+    "description": "a form",
+    "organisationId": "0101010101010",
+    "formsAppEnvironmentId": 1,
+    "elements": [],
+    "isAuthenticated": false,
+    "submissionEvents": []
+  },
+  "history": []
+}
+```
+
+## getFormApprovalFlowInstanceById
```

Comment:
Should change this one as well.

Reply:
Can we also add dot points to the top like the forms class docs for each function.